include/ajax.tasks.php

```diff
@@ -655,7 +616,7 @@ function massProcess($action, $w=null)  {
             $inc = 'task-status.tmpl.php';
             $info[':action'] = "#tasks/mass/$action";
             $info['status'] = $info['status'] ?: 'closed';
-            $perm = $action = '';
+            $perm = '';
             switch ($info['status']) {
```

Comment:
We need to make sure we don't clear $action here because we use it to get the correct verb for the success message.